ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -0,0 +1,249 @@
+---
+    - name: Packages                   | Add Nginx repository
+      yum_repository:
+        name: nginx
+        description: nginx repo
+        baseurl: http://nginx.org/packages/mainline/centos/7/$basearch/
+        gpgcheck: no
+        enabled: yes
+
+    - name: Packages                   | Add PMM2 Server YUM repository
+      yum_repository:
+        name: pmm2-laboratory
+        description: PMM Server YUM repository - x86_64
+        baseurl: https://repo.percona.com/pmm2-components/yum/{{ pmm2_server_repo }}/7/RPMS/x86_64/
+        gpgcheck: yes
+        enabled: no
+        gpgkey: file:///etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
```

Comment:
Can we use percona-release tool there?

Reply:
not for now, because percona-release does not support pmm2-components repos for now